src/lib/Lib/ConfigImpl.cpp

```diff
@@ -83,6 +84,39 @@ ConfigImpl(
     }
     settings_.configDir = files::makeDirsy(files::normalizePath(settings_.configDir));
 
+    if (settings_.libCxxPaths.empty())
```

Comment:
setupAddonsDir in Addons.hpp does that for addons. Does one of these designs need to be better?